driver/bpf/plumbing_helpers.h

```diff
@@ -617,6 +618,18 @@ static __always_inline void call_filler(void *ctx,
 	if (!settings)
 		return;
 
+	/* Check if syscall was successful */
+	// Odd evt types are used for exit events;
+	// Note that for non-syscall event types we always use enter events
+	if (evt_type & 1 && settings->drop_failed)
```

Comment:
Can we use the `PPME_IS_EXIT()` macro here? Same in all other places

Reply:
why do we implement it here and not in the `sys_exit` dedicated code? we should be able to access the return value directly with `bpf_syscall_get_retval(ctx)`

Reply:
I am using it now.

Reply:
Done!